src/components/Body/Button.js

```diff
@@ -0,0 +1,17 @@
+import Button from 'react-bootstrap/Button';
+
+function AddToCartButton(props) {
+    return (
+        <Button
+            variant={props.variant}
+            id={props.id}
+            onClick={(e) => {
+                props.handleCard(e.target.id);
+            }}
```

Comment:
Hej pa ovo je React a ne VanillaJS 😄 :

```suggestion
            onClick={() => props.handleCard(props.id)}
```